src/App/backend/src/Altinn.App.Core/Internal/LocaltestValidation.cs

```diff
@@ -75,7 +87,30 @@ protected override async Task ExecuteAsync(CancellationToken stoppingToken)
                         {
                             _logger.LogInformation("Localtest version: {Version}", version);
                             if (version >= 2)
```

Comment:
This change would be a version bump of localtest, so registration should be available at n+1